### PR TITLE
Update Japanese translation from v1.17 to v1.19.

### PR DESCRIPTION
**Pull request by kaleidot725**
Update Japanese translation from v1.17 to v1.19

### CODE REVIEW

File: README.jp.md [hunk 1]
```diff
@@ -233,6 +237,77 @@ scrcpy -Nr file.mkv
 
 [パケット遅延のバリエーション]: https://en.wikipedia.org/wiki/Packet_delay_variation
 
+#### v4l2loopback
```
Comment (rom1v) [inline README.jp.md:240]: In the main README, `Recording` (`録画`) has been incremented by one level, and a title `Capture` has been added:
```diff
-### Recording
+### Capture
+
+#### Recording
…
+#### v4l2loopback
```

Could you add this change please (in particular translate `Capture`)?

**Review by rom1v — COMMENTED** [review]
Thank you :+1: Just a little comment inline.

**Comment by kaleidot725** [comment]
Added `キャプチャ` heading to README.jp.md, because `Capture` translates to `キャプチャ`.  (740e165aab5e60201f2eb47e1cef72aba27d07f5.)
Please check it.

**Comment by rom1v** [comment]
Merged: 790f04e91faa14c5ebd2b987530d1e3db080e682

**Pull request closed without merging**